dockers/app/requirements.pip3.original

```diff
@@ -0,0 +1,44 @@
+APScheduler==3.3.1
+asn1crypto==0.24.0
+bcrypt==3.1.4
+certifi==2018.4.16
+cffi==1.10.0
+chardet==3.0.4
+click==6.7
+cryptography==2.1.4
+Flask==0.12.2
+Flask-Login==0.4.1
+Flask-SocketIO==2.8.6
+gevent==1.2.2
+graphyte==1.4
+greenlet==0.4.13
+idna==2.7
+iniparse==0.4
+itsdangerous==0.24
+Jinja2==2.9.6
+libvirt-python==4.4.0
+lxml==4.2.2
+MarkupSafe==1.0
+netaddr==0.7.19
+numpy==1.16.4
+pandas==0.24.2
+paramiko==2.4.2
+pem==18.2.0
+pexpect==4.6.0
+ptyprocess==0.6.0
+pyasn1==0.4.2
+pycparser==2.18
+PyNaCl==1.2.1
+pynpm==0.1.1
+pyOpenSSL==17.5.0
+python-dateutil==2.8.0
+python-engineio==3.8.1
+python-socketio==4.1.0
+pytz==2019.1
+requests==2.19.1
+rethinkdb==2.3.0.post6
+six==1.11.0
+tzlocal==1.5.1
+urllib3==1.22
+Werkzeug==0.11.15
+xmltodict==0.11.0
```

Comment:
Is this file necessary? It could be fetched through the git history

Reply:
tests for upgrading